history-network.md

```diff
@@ -135,15 +113,13 @@ The chain history DHT stores the following data items:
 * Block bodies
 * Receipts
 * Header epoch accumulators
-* Header master accumulator
 
-Each of these data items are represented as a key-value pair. Denote the key for a data item by `content-key`. Denote the value for an item as `content`.
+Each of these data items are represented as a key-value pair. 
 
-All `content_key` values are encoded and decoded as an [`SSZ Union`](https://github.com/ethereum/consensus-specs/blob/dev/ssz/simple-serialize.md#union) type.
```

Comment:
The portal wire and state specs also mention the SSZ Union in similar way. It would be good to drop that also there for consistency.

Reply:
Doing this in  separate PR